src/main/java/edu/ucsb/cs156/frontiers/repositories/InstructorRepository.java

```diff
@@ -0,0 +1,14 @@
+package edu.ucsb.cs156.frontiers.repositories;
+
+import edu.ucsb.cs156.frontiers.entities.Instructor;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ * Repository for Instructor entities.
+ * Provides CRUD operations for the Instructor entity using email as the primary key.
+ */
+@Repository
+public interface InstructorRepository extends CrudRepository<Instructor, String> {
```

Comment:
@RubenAlvarezGJ left this feedback in his code review:

```
You need to change InstructorRepository.java to allow for email sorting. You can do this by changing lines 12-14 to "public interface AdminRepository extends JpaRepository<Admin, String> {}" and adding the import "import org.springframework.data.jpa.repository.JpaRepository;" See https://github.com/ucsb-cs156-s25/proj-frontiers-s25-09/pull/53 for reference if needed. Also add a dokku deployment for testing please.
```

Please respond to that code review feedback.